app/services/publish_manual_service.rb

```diff
@@ -8,6 +10,8 @@ def initialize(dependencies)
 
   def call
     if versions_match?
+      fetch_tags
+      update_manual_with_tags
```

Comment:
Could there be an advantage to persisting a manual with its new tags regardless of the outcome of the publishing steps?